### ISSUE TITLE
Inlay hints command has title "Inline", which is confusing

### ISSUE THREAD

**Issue by meganrogge** [microsoft/vscode#247435]
Found by @jooyoungseo

This command should be `Read Line with Inlay Hints` vs `Read Line with Inline Hints`

<img width="675" alt="Image" src="https://github.com/user-attachments/assets/d7a74247-2acd-42a9-bcc1-f410ffb8d85c" />

<img width="1040" alt="Image" src="https://github.com/user-attachments/assets/818cd27c-0bf5-4231-8424-a45715c9d75b" />

**Comment by meganrogge** [comment]
Thanks for fixing!
